src/apm.ts

```diff
@@ -1,9 +1,22 @@
 import { EventEmitter } from 'events';
 import type { Callback } from './utils';
+import { Connection } from './cmap/connection';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
what's up with this comment?

Reply:
I'm not entirely sure, we have a two other cases where this comes up
https://github.com/mongodb/node-mongodb-native/blob/ca16d554a6d40cb3cbf35164ed5c94de10537bc3/src/operations/collections.ts#L6-L8
https://github.com/mongodb/node-mongodb-native/blob/ca16d554a6d40cb3cbf35164ed5c94de10537bc3/src/operations/create_collection.ts#L10-L12
Despite the type being used its marked as unused for some reason

Reply:
oh I think this was from before we upgraded prettier, you can probably just remove the lines everywhere

Reply:
Oh yea you are right those were fine to remove, but this apm one still reports unused var despite it being used. Maybe because it is being used as a type annotation for a `this` on the function in instrument, which is kind of a fake param to type your function's this binding, not sure though